packages/studio-plugin/tests/sourcefiles/LayoutFile.test.ts

```diff
@@ -87,7 +87,7 @@ describe("getLayoutState", () => {
   it("correctly parses CSS imports", () => {
     const result = getLayoutState("BasicLayout");
     expect(result.value.cssImports).toEqual([
-      "./index.css",
+      expect.stringContaining("index.css"),
```

Comment:
nit: in this test and the one for PageFile, can we check the exact path that's computed since that's the key part of this change set?

Reply:
done!